PowerShell.Common.props

```diff
@@ -94,7 +94,7 @@
     <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
 
     <TargetFramework>netcoreapp2.0</TargetFramework>
-    <RuntimeFrameworkVersion>2.0.0</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>2.0.4-servicing-25831-01</RuntimeFrameworkVersion>
```

Comment:
The package that was tested was "25824-03".
This one, in this change, is just newer?

Reply:
Yes, it's a week newer. Produced on 10/31

Reply:
I see 2.0.4 here and 2.0.2 in Build.psm1 and global.json - could you please clarify the difference?

Reply:
The 2.0.2 in global.json indicates the version of the .NET Core SDK, see [global.json reference](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json). The `2.0.4-servicing` here is the version of dotnet core runtime the build will target.

`dotnet --info` shows 2 versions: .NET Command Line Tools version and the default .NET Core framework version. Currently the 2.0.2 .NET Core SDK still target .NET Core runtime 2.0.0 by default.
```
.NET Command Line Tools (2.0.2)

Product Information:
 Version:            2.0.2
 Commit SHA-1 hash:  a04b4bf512
...
Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d
```